### DISCUSSION TITLE
Seprate CAZ into Canada, Australia and New Zealand during the simulations

### DISCUSSION THREAD

**Discussion by KeLiu7** [magpiemodel/magpie#538] [Q&A]
Dear MAgPIE community, 

Is it possible to separate CNZ to Canada, New Zealand and Australia during the simulations in MAgPIE? Can I just change the region code to realize this? Thanks for your help in advance.

**Comment by tscheypidi** [comment]
Hi Ke,

the input data has to be newly preprocessed in order to run MAgPIE with a different spatial resolution. That step currently requires some assistance from PIK side. I will write you a mail to discuss the next steps.

Jan
